lmcache/integration/vllm/vllm_v1_adapter.py

```diff
@@ -941,9 +942,13 @@ def build_connector_meta(
             if req_meta is not None:
                 meta.add_request(req_meta)
 
-        for request in scheduler_output.scheduled_cached_reqs:
-            request_tracker = self._request_trackers[request.req_id]
-            request_tracker.update(request)
+        cached_reqs = scheduler_output.scheduled_cached_reqs
+        for i, req_id in enumerate(cached_reqs.req_ids):
+            request_tracker = self._request_trackers[req_id]
+            new_token_ids = cached_reqs.new_token_ids[i]
```

Comment:
These fields of `cached_reqs` could not be found in the previous supported vllm.

Any throughts?